src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/Dashboard.stories.tsx

```diff
@@ -183,6 +184,11 @@ const DashboardWrapper = (props: DashboardWrapperProps) => {
     user: user,
   };
 
+  const mockedRemovalTimeEstimates = scanData.results.map((scan) => ({
+    d: scan.data_broker,
+    t: faker.number.float({ min: 0, max: 200 }),
+  }));
```

Comment:
Maybe not relevant now that the flags are disabled by default, but it might be fun/more realistic to have some "Unknown"s on the dashboard as well:

```suggestion
  })).filter(() => Math.random() < 0.1);
```

Reply:
Yup: [293c1e7](https://github.com/mozilla/blurts-server/pull/5189/commits/293c1e71dbe5817c4b009b3bdbda831ba1b4c54f)!